src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -330,6 +359,7 @@ ss::future<response_ptr> create_topics_handler::handle(
         append_topic_configs(ctx, response);
     }
 
+    log_topic_status(std::move(c_res));
```

Comment:
questio: I know it's not used after this point, but any reason to not pass it by const ref?